src/main/java/io/airlift/junit/InitializeLogging.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.junit;
+
+import io.airlift.log.Level;
+import io.airlift.log.Logging;
+import org.junit.jupiter.api.extension.Extension;
+
+public class InitializeLogging
+        implements Extension
+{
+    static {
+        Logging logging = Logging.initialize();
+
+        // disable logging from testcontainers by default
+        logging.setLevel("tc", Level.OFF);
+        logging.setLevel("org.testcontainers", Level.OFF);
```

Comment:
Make this WARN

Reply:
Which errors? As far as I can tell, those will propagate all the way up the stack and cause tests to fail with an exception that will get printed by the testing framework.